Feedback/Model/ResourceModel/Feedback.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Tasks\Feedback\Model\ResourceModel;
+
+use Magento\Framework\Model\ResourceModel\Db\AbstractDb;
+
+/**
+ * Feedback Resource Model
+ *
+ * @package Tasks\Feedback\Model\ResourceModel
+ */
+class Feedback extends AbstractDb
+{
+    /**
+     * Main table name
+     */
+    const MAIN_TABLE = 'feedback_form';
```

Comment:
Could have been a better name like this
```suggestion
    const FEEDBACK_DETAILS_TABLE = 'feedback_form';
```